ros2topic/ros2topic/api/__init__.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.node.strategy import NodeStrategy
+
+HIDDEN_TOPIC_PREFIX = '_'
+
+
+def get_topic_names_and_types(*, node, include_hidden_topics=False):
```

Comment:
Those are literals, there is no way that `get_topic_names_and_types(node, ...)` is ambiguous, unless the node parameter name is bad, but that's it's own problem.

I could be convinced that this would be a good thing to do for literals, but it is still the case that this code is in direct contradiction to most of our existing code. If this is the "new way forward" it needs to be documented in our developer guide after we discuss it as a team.